pythonscript/_godot.pyx

```diff
@@ -46,8 +47,8 @@ cdef api godot_pluginscript_language_data *pythonscript_init():
 
     # TODO
     # Redirect stdout/stderr to have it in the Godot editor console
-    # if _setup_config_entry("python_script/io_streams_capture", True):
-    #     enable_capture_io_streams()
+    #if _setup_config_entry("python_script/io_streams_capture", True):
```

Comment:
Don't you think should re-enable this option ?

Reply:
Done!